### PR TITLE
fix(MWPW-138316):scrolling inside an iframe is disabled.

### PR DESCRIPTION
**Pull request by sharath-kannan**
* scrolling inside an iframe is disable as is causes multiple scrollbars to be visible.

Resolves:[MWPW-138316](https://jira.corp.adobe.com/browse/MWPW-138316)

**Test URLs:**
- Before:https://main--cc--adobecom.hlx.live/es/creativecloud/animation/discover?martech=off#animate
- After: https://main--cc--adobecom.hlx.live/es/creativecloud/animation/discover?martech=off&milolibs=MWPW-138316--milo--sharath-kannan#animate

view the below file to get a clear picture of the issue.
Screenshot 2024-01-16 at 2 52 29 PM 2

### CODE REVIEW

**Comment by JasonHowellSlavin** [comment]
It might be good to hunt down some other uses of the iframe block for other consumers to see how this change effects those. I will look for some for bacom.

File: libs/blocks/modal/modal.css [hunk 1]
```diff
@@ -25,6 +25,10 @@
   z-index: 103;
 }
 
+.dialog-modal.commerce-frame .fragment,.dialog-modal.commerce-frame .section {
```
Comment (meganthecoder) [inline libs/blocks/modal/modal.css:28]: One item per line
```suggestion
.dialog-modal.commerce-frame .fragment,
.dialog-modal.commerce-frame .section {
```

**Review by meganthecoder — COMMENTED** [review]
I like that you've made the solution specific to commerce modals so that other iframes aren't affected. That being said, I question the "trivial" label. If this will impact production, it seems like it should be verified by a QE.

**Comment by sharath-kannan** [comment]
> I like that you've made the solution specific to commerce modals so that other iframes aren't affected. That being said, I question the "trivial" label. If this will impact production, it seems like it should be verified by a QE.

Thanks for reviewing the PR, i have committed your suggestion and changed the label to "verified".

**Comment by Snehayt** [comment]
Verified the scrollbar.Looks fine now.
https://main--cc--adobecom.hlx.live/es/creativecloud/animation/discover?martech=off&milolibs=MWPW-138316--milo--sharath-kannan#animate
https://main--cc--adobecom.hlx.live/es/creativecloud/animation/discover?martech=off&milolibs=MWPW-138316--milo--sharath-kannan#animate

image
image
image

**Review by meganthecoder — APPROVED** [review]

**Comment by Ruchika4** [comment]
Hi @sharath-kannan , Thanks for fixing this. Looks good. I verified on homepage also as they use the commerce modal too. I see a little more white space at the bottom with this update added at the modal
Page: https://main--homepage--adobecom.hlx.page/homepage/index-loggedout?milolibs=MWPW-138316--milo--sharath-kannan#mini-plans-web-cta-creative-cloud-card
Before:
Screen Shot 2024-01-24 at 10 57 34 AM
After:
Screen Shot 2024-01-24 at 11 00 20 AM

It's minor 5-6px increase at the bottom. However tagging @Blainegunn  and @vgoodric  who initially had fixed the height as part of this ticket https://jira.corp.adobe.com/browse/MWPW-135674 to confirm if this is ok from homepage perspective.

**Comment by sharath-kannan** [comment]
> Hi @sharath-kannan , Thanks for fixing this. Looks good to me. I verified on homepage also as they use the commerce modal too. I see a little more white space at the bottom with this update added at the modal Page: https://main--homepage--adobecom.hlx.page/homepage/index-loggedout?milolibs=MWPW-138316--milo--sharath-kannan#mini-plans-web-cta-creative-cloud-card Before: Screen Shot 2024-01-24 at 11 00 20 AM
> 
> After: Screen Shot 2024-01-24 at 10 57 34 AM
> 
> It's minor 5-6px increase at the bottom. However tagging @Blainegunn and @vgoodric who initially had fixed the height as part of this ticket https://jira.corp.adobe.com/browse/MWPW-135674 to confirm if this is ok from homepage perspective.

Hi @Ruchika4 
The page by itself when viewed in a separate tab has the white space, is this the white space  you are referring to?

image

**Comment by Ruchika4** [comment]
> https://main--homepage--adobecom.hlx.page/homepage/index-loggedout?milolibs=MWPW-138316--milo--sharath-kannan#mini-plans-web-cta-creative-cloud-card

I verified again on this page https://main--homepage--adobecom.hlx.page/homepage/index-loggedout#mini-plans-web-cta-photoshop-card and don't see the spacing issue. As the issue is not reproducible for me now approving the PR

**Review by Ruchika4 — APPROVED** [review]

**Review by vgoodric — APPROVED** [review]

**Pull request merged**